server/src/main/java/edu/jhu/server/Game.java

```diff
@@ -41,124 +42,142 @@ public boolean isStarted() {
+    this.secretCards = CardShuffler.shuffleAndDealCards(players);
+    this.board.initialize();
   }
 
   public void addPlayer(Player player) {
     players.add(player);
     player.setGame(this);
   }
 
+  public void sendTurnNotification() {
+    Player player = players.get(currentTurnIndex);
+
+    JSONArray validMoves = new JSONArray();
+    board.getValidMoves(player.getSuspect()).forEach(loc -> validMoves.put(loc.toString()));
+
+    JSONObject notification = new JSONObject();
+    notification.put("eventType", "TURN_NOTIFICATION");
+    notification.put("playerTag", player.getTag());
+    notification.put("validMoves", validMoves);
+
+    notifyPlayers(notification);
+  }
+
   public void notifyPlayers(JSONObject event) {
     for (Player player : players) {
       player.sendEvent(event);
     }
   }
-  
+
   private JSONObject makeChatMessage(String body) {
-	  JSONObject chat = new JSONObject();
-	  chat.put("eventType", "CHAT_NOTIFICATION");
-	  chat.put("author", "Game");
-	  chat.put("body", body);
-	  return chat;
-  }
-  
-  private JSONObject makeInvalidRequestMessage(String player, String reason){
-	  JSONObject invalidRequest = new JSONObject();
-	  invalidRequest.put("eventType", "INVALID_REQUEST_NOTIFICATION");
-	  invalidRequest.put("author", "Game");
-	  invalidRequest.put("reason", reason);
-	  invalidRequest.put("player", player);
-	  return invalidRequest;
-  }
-  
-  private JSONObject makeMoveNotification(Suspect suspect, ILocation location){
-	  JSONObject move = new JSONObject();
-	  move.put("eventType", "MOVE_NOTIFICATION");
-	  return move;
-  }
-  
-  
-  
+    JSONObject chat = new JSONObject();
+    chat.put("eventType", "CHAT_NOTIFICATION");
+    chat.put("author", "Game");
+    chat.put("body", body);
+    return chat;
+  }
+
+  private JSONObject makeInvalidRequestMessage(String player, String reason) {
+    JSONObject invalidRequest = new JSONObject();
+    invalidRequest.put("eventType", "INVALID_REQUEST_NOTIFICATION");
+    invalidRequest.put("author", "Game");
+    invalidRequest.put("reason", reason);
+    invalidRequest.put("player", player);
+    return invalidRequest;
+  }
+
+  private JSONObject makeMoveNotification(Suspect suspect, ILocation location) {
+    JSONObject move = new JSONObject();
+    move.put("eventType", "MOVE_NOTIFICATION");
+    return move;
+  }
+
   private void provideEvidence(CaseFile casefile, Player suggester) {
-	  Player playerWithEvidence = null;
-	  for (Player player : players) {
-		  if(player.getTag() != suggester.getTag() && 
-			(player.hasCard(casefile.getRoom()) || 
-			player.hasCard(casefile.getSuspect()) ||
-			player.hasCard(casefile.getWeapon()))) {
-			  playerWithEvidence = player;
-			  break;
-		  }  
-	  }
-	  if (playerWithEvidence == null) {
-		  JSONObject chat = makeChatMessage("Nobody could provide evidence against this suggestion!");
-		  handleEvent(chat);
-	  }
-  }
-  
+    Player playerWithEvidence = null;
+    for (Player player : players) {
```

Comment:
Technically, this should start with the player immediately after the suggester and continue through the players, wrapping around. Instead of always starting with player 1.